test/runtests.jl

```diff
@@ -60,15 +61,29 @@ macro includetests(testarg...)
     end
 end
 
-gl = global_logger()
-level = get(ENV, "PS_LOG_LEVEL", "Error")
-log_level = get(LOG_LEVELS, level, nothing)
-if log_level == nothing
-    error("Invalid log level $level: Supported levels: $(values(LOG_LEVELS))")
+function get_logging_level(env_name::String, default)
+    level = get(ENV, env_name, default)
+    log_level = get(LOG_LEVELS, level, nothing)
+    if log_level == nothing
+        error("Invalid log level $level: Supported levels: $(values(LOG_LEVELS))")
+    end
+
+    return log_level
 end
-global_logger(ConsoleLogger(gl.stream, log_level))
 
-# Testing Topological components of the schema
-@testset "Begin PowerSystems tests" begin
-    @includetests ARGS
+console_level = get_logging_level("PS_LOG_LEVEL", "Error")
+file_level = get_logging_level("PS_CONSOLE_LOG_LEVEL", "Info")
```

Comment:
Is the environment name here right? 

```
file_level = get_logging_level("PS_CONSOLE_LOG_LEVEL", "Info")
```

We getting a `PS_CONSOLE_LOG_LEVEL` but storing the value in variable called `file_level`.

Reply:
Doh.  Fixed, thanks.

Reply:
If a user sets these environment variables, at the moment, they will only take effect when they run tests. If I want to create a file logger when I'm using `PowerSystems` to parse some file, I wouldn't get these logging levels used unless I implemented the same as above, right?

Reply:
That is correct.  Logging has to be configured by the application, whatever that is.  I'm guessing that that is going to be some higher-level entity, like a consumer of PowerSimulations, right?  I'm totally open to making this easier, but I probably need to better understand the use cases.

Reply:
Resolved in aa166cd